docker/centos8/Dockerfile

```diff
@@ -0,0 +1,18 @@
+FROM rockylinux:8.5.20220308
```

Comment:
We should rename the occurrences of centos8 to be rockylinux8. (build-rpms.yml and docker/centos8/Dockerfile)